Documentation/CRDs/Shared-Filesystem/ceph-filesystem-crd.md

```diff
@@ -135,13 +135,16 @@ spec:
     # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
     peers:
       secretNames:
-        - secondary-cluster-peer
+      # Add the secret name if it already exists else specify the empty list here.
     # specify the schedule(s) on which snapshots should be taken
     # see the official syntax here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-schedules
     snapshotSchedules:
       - path: /
         interval: 24h # daily snapshots
-        startTime: 11:55
+        startTime: 2022-07-15T11:55:00
+        # The startTime should be mentioned in the format YYYY-MM-DDTHH:MM:SS
+        # If startTime is skipped, then by default the start time is considered as last midnight.
```

Comment:
It's UTC, right?
```suggestion
        # If startTime is not specified, then by default the start time is considered as midnight UTC.
```

Reply:
@travisn yep I just confirmed with the cephfs team in IRC  #cephfs that its UTC : `<mchangir> gauravsitlani, if start time doesn't have time zone info when creating the schedule, it is assumed to be in UTC; else, the timezone in the start date string is used and then mapped to UTC` 
I'll update the suggested changes here.

Reply:
To have this updated in CephFS documentation the following tracker has been opened : https://tracker.ceph.com/issues/56730